win32ss/gdi/gdi32/objects/font.c

```diff
@@ -2062,32 +2062,44 @@ RemoveFontResourceA(LPCSTR lpFileName)
 }
 
 /*
- * @unimplemented
+ * @implemented
  */
 BOOL
 WINAPI
 RemoveFontResourceExA(LPCSTR lpFileName,
                       DWORD fl,
-                      PVOID pdv
-                     )
+                      PVOID pdv)
 {
     NTSTATUS Status;
     LPWSTR lpFileNameW;
+    BOOL result;
 
-    /* FIXME the flags */
-    /* FIXME the pdv */
-    /* FIXME NtGdiRemoveFontResource handle flags and pdv */
+    if (fl & ~(FR_PRIVATE | FR_NOT_ENUM))
+    {
+        SetLastError(ERROR_INVALID_PARAMETER);
+        return FALSE;
+    }
 
-    Status = HEAP_strdupA2W ( &lpFileNameW, lpFileName );
-    if (!NT_SUCCESS (Status))
-        SetLastError (RtlNtStatusToDosError(Status));
-    else
+    _SEH2_TRY
+    {
+        Status = HEAP_strdupA2W(&lpFileNameW, lpFileName);
+    }
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
     {
+        DPRINT1("Exception!\n");
+        _SEH2_YIELD(return FALSE);
```

Comment:
Shouldn't this line be `Status = ...;` instead?

Reply:
Fixed.